core/src/components/item/item.scss

```diff
@@ -72,6 +72,8 @@
   --color-focused: var(--color);
   --color-hover: var(--color);
   --ripple-color: currentColor;
+  --background-autofill: rgb(232, 240, 254);
+  --color-autofill: black;
```

Comment:
I believe this should be `var(--color)`. Not sure about the background though, would be good to discuss more internally.